src/tests/ui/planner/workitem-detailpage.ts

```diff
@@ -0,0 +1,13 @@
+import { ElementFinder, $ } from 'protractor';
```

Comment:
We can remove this. I don't see where it's being used.

Reply:
done!